modules/rosa-hcp-architecture.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa_architecture_sub/rosa-architecture-models.adoc
+
+[id="rosa-hcp-architecture_{context}"]
+= ROSA with HCP architecture
+
+In ROSA with HCP, the ROSA service hosts a highly-available, single-tenant OpenShift control plane. The hosted control plane is deployed across 3 availability zones with at least 2 API server instances and three etcd instances. It is only accessible from the customer's VPC subnets. Customers access the hosted control plane through AWS PrivateLink. Customers can add an optional internet-facing API server endpoint to their ROSA with HCP cluster.
```

Comment:
I'd suggest reframing the following sentence as proposed below because the default today is that HCP clusters give internet-facing API server endpoint.

_"Customers can add an optional internet-facing API server endpoint to their ROSA with HCP cluster"_

"Customers can create their ROSA with HCP cluster with or without internet-facing API Server."